apps/emqx_exhook/priv/protos/exhook.proto

```diff
@@ -368,6 +369,8 @@ message ConnInfo {
   string proto_ver = 7;
 
   uint32 keepalive = 8;
+
+  uint32 peerport = 14;
```

Comment:
Q: Why does it need to be set to 14 here instead of 9?

Reply:
A: peerhost and peerport are two closely related fields and exist in both `conninfo` and `clientinfo`, with different field numbers.
For consistency, peerhost+10 is used as the peerport number.